### PR TITLE
Yal 941 yal whatsapp bot prd 2 d value error

### PR DESCRIPTION
**Pull request by Buhle79**

### CODE REVIEW

File: yal/utils.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def normalise_phonenumber(phonenumber):
             assert phonenumbers.is_valid_number(pn)
             return phonenumbers.format_number(pn, phonenumbers.PhoneNumberFormat.E164)
     except (phonenumbers.phonenumberutil.NumberParseException, AssertionError):
-        raise ValueError("Invalid phone number")
+        raise ValueError("Invalid phone number {}".format(phonenumber))
```
Comment (erikh360) [inline yal/utils.py:78]: Can we use a f-string here?

File: yal/tests/test_utils.py [hunk 2]
```diff
@@ -122,3 +123,21 @@ def test_is_integer():
 
     obj = 1234
     assert utils.is_integer(obj)
+
+
+class PhoneNumberTest(unittest.TestCase):
```
Comment (erikh360) [inline yal/tests/test_utils.py:128]: Are we inheriting from `unittest.TestCase` to use `assertRaises`?

**Review by erikh360 — COMMENTED** [review]
Nice, I have one comment and one question.

Also, are we just adding this to debug? It might not be a great idea to send PII to sentry?

File: yal/utils.py (hunk 1 above)
Comment (Buhle79) [reply yal/utils.py:78]: Yes, I've added user phone number for debugging here.

File: yal/tests/test_utils.py (hunk 2 above)
Comment (Buhle79) [reply yal/tests/test_utils.py:128]: No i like to group all related test under a class

**Review by erikh360 — APPROVED** [review]

**Pull request merged**